docs/raph-rover/documentation/getting-started.mdx

```diff
@@ -0,0 +1,347 @@
+import FlexTableItem from '@site/src/components/FlexTableItem';
+import CenterContent from '@site/src/components/CenterContent';
+
+import Support from '@site/docs/raph-rover/partial/_support.mdx';
+
+<ImageZoom
+  loading="eager"
+  src="/img/robots/raph/raph-rover.webp"
+  alt="Raph Rover isometric render"
+  width="3000"
+  height="2250"
+  figStyle={{
+    width: 500,
+    paddingBottom: '2rem',
+  }}
+/>
+
+# Getting started with Raph Rover
+
+---
+
+## Unboxing
+
+Currently, Raph Rover comes only in assembled version. All essential components
+are pre-installed and ready for use out of the box.
+
+Before attempting to power on the rover, unbox the robot from protective foam
+package and verify that all accessories listed in the package contents are
+present and undamaged.
+
+<CenterContent>
+
+| Item                       | Count |
+| -------------------------- | :---: |
+| Raph Rover                 |   1   |
+| Raph Rover battery pack    |   2   |
+| Raph Rover battery charger |   1   |
+| Li-Ion charger 6S 4A       |   1   |
+| Tools                      |   1   |
+
+</CenterContent>
+
+{/* TODO: ADD packed Raph render */}
+
+### Rover buttons
+
+Raph rover is equipped with 2 physical buttons that we will reference in this
+guide. Power button is located next to the left battery slot. Second, control
+button is located at the rear of the robot.
+
+<ThemedImageZoom
+  alt="Diagram with arrows pointing to the buttons of Raph Rover, showing power and multifunctional buttons"
+  sources={{
+    light: useBaseUrl(
+      '/img/robots/raph/getting-started/raph_buttons_black.webp',
+    ),
+    dark: useBaseUrl(
+      '/img/robots/raph/getting-started/raph_buttons_white.webp',
+    ),
+  }}
+  width="3000"
+  height="2121"
+  figStyle={{
+    width: 600,
+  }}
+/>
+
+<CenterContent>
+
+| Button                                    | Placement |
+| ----------------------------------------- | :-------: |
+| <font color="2c71bf"> **POWER** </font>   | **Left**  |
+| <font color="c07b2c"> **CONTROL** </font> | **Back**  |
+
+</CenterContent>
+
+Each button has a different function. What it does depends on the robot's state.
+
+<CenterContent>
+
+| Condition |                  Button                   |        Function         |  Action   |
+| --------- | :---------------------------------------: | :---------------------: | :-------: |
+| Any       |  <font color="2c71bf"> **POWER** </font>  |  Check battery charge   |   Click   |
+| Rover off |  <font color="2c71bf"> **POWER** </font>  |         Turn on         | Hold - 3s |
+| Rover on  |  <font color="2c71bf"> **POWER** </font>  |        Turn off         | Hold - 5s |
+| Rover on  | <font color="c07b2c"> **CONTROL** </font> | Toggle maintenance mode |   Click   |
+| Rover on  | <font color="c07b2c"> **CONTROL** </font> |    Calibrate servos     | Hold - 3s |
+
+</CenterContent>
+
+## Connecting batteries
+
+Each Raph Rover is equipped with two dedicated 24V Li-Ion battery packs. As long
```

Comment:
Consider using 6S instead of 24V. IMO 24V is often misleading. It's worth mentioning somewhere that the actual voltage during operation is between 18 V and 25.2 V